extension/observer/k8sobserver/README.md

```diff
@@ -75,4 +75,169 @@ All fields are optional.
 | observe_ingresses | bool | `false` | Whether to report observer k8s.ingress endpoints.|  
 
 More complete configuration examples on how to use this observer along with the `receiver_creator`,
-can be found at the [Receiver Creator](../../../receiver/receivercreator/README.md)'s documentation.
\ No newline at end of file
+can be found at the [Receiver Creator](../../../receiver/receivercreator/README.md)'s documentation.
+
+### Setting up RBAC permissions
+
+When using the `serviceAccount` `auth_type`, the service account of the pod running the agent needs to have the required permissions to
+read the K8s resources it should observe (i.e. pods, nodes, services and ingresses).
+Therefore, the service account running the pod needs to have the required `ClusterRole` which grants it the permission to
+read those resources from the Kubernetes API. Below is an example of how to set this up:
+
+1. Create a `ServiceAccount` that the collector should use.
+
+```bash
+<<EOF | kubectl apply -f -
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: otelcontribcol
+  name: otelcontribcol
+EOF
+```
+
+2. Create a `ClusterRole`/`ClusterRoleBinding` that grants permission to read pods, nodes, services and ingresses.
```

Comment:
I suppose using just a `Role` instead of `ClusterRole` is also an option if the user only want to observe e.g. pods in a specific namespace?

Reply:
Sorry for the late reply - right now, the k8sobserver works only with `ClusterRoles`, but I'm working on an issue (#9401) to also support the use of namespaced resources and thus also be usable with `Roles` instead of `ClusterRoles`

Reply:
Edit: I just realized that the issue i linked in the comment above is for the `k8sclusterreceiver`, and not the `k8sobserver` - However, looking at the code of the k8sobserver, it seems like the informer being created here also is currently always attempting to observe all namespaces. so it might make sense to also do something similar as in #9401 for this component.